docs/src/components/theme-designer/preview.astro

```diff
@@ -21,9 +21,9 @@ const codeBg = light ? '--sl-color-gray-6' : '--sl-color-gray-5';
 <div class="preview-card" {...attrs}>
 	<h4>{light ? labels.lightMode : labels.darkMode}</h4>
 	<p>
-		{labels.bodyText}
-		<span class="link" style={`color:var(${linkColor})`}>{labels.linkText}</span>
-		<span class="dim">{labels.dimText}</span>
+		{labels.bodyText}&nbsp;
+		<span class="link" style={`color:var(${linkColor})`}>{labels.linkText}</span>&nbsp;
+		<span class="dim">{labels.dimText}</span>&nbsp;
```

Comment:
Ah, this is due to how Astro started compressing HTML somewhat aggressively in 3.0 — https://github.com/withastro/compiler/issues/852

I’d suggest this fix:
```suggestion
		{labels.bodyText}{' '}
		<span class="link" style={`color:var(${linkColor})`}>{labels.linkText}</span>{' '}
		<span class="dim">{labels.dimText}</span>{' '}
```
We don’t need these to be non-breaking spaces, just force Astro into generating a space there.

Reply:
Thank you very much!